src/lib/webpack/webpack-base-config.js

```diff
@@ -57,26 +42,19 @@ export default env => {
 	env.pkg = readJson(resolve(cwd, 'package.json')) || {};
 	env.manifest = readJson(src('manifest.json')) || {};
 
-	return createConfig.vanilla([
+	return group([
 		setContext(src('.')),
-		entryPoint(resolve(__dirname, './entry')),
-		setOutput({
-			path: resolve(cwd, env.dest || 'build'),
-			publicPath: '/',
-			filename: 'bundle.js',
-			chunkFilename: '[name].chunk.[chunkhash:5].js'
-		}),
 
 		customConfig({
 			resolve: {
 				modules: [
 					'node_modules',
-					resolve(__dirname, '../../node_modules')
+					resolve(__dirname, '../../../node_modules')
```

Comment:
We should just be using NODE_PATH for this wherever possible. Actually, anywhere we have node_modules references for resolving, we should add NODE_PATH. That'll make us future proof.